src/cdk-experimental/scrolling/virtual-scroll-viewport.ts

```diff
@@ -245,7 +246,28 @@ export class CdkVirtualScrollViewport implements OnInit, OnDestroy {
     }
   }
 
-  /** Sets the scroll offset on the viewport. */
+  /**  Scrolls to the offset on the viewport. */
+  scrollToOffset(offset: number, behavior: ScrollBehavior = 'auto') {
+    const viewportElement = this.elementRef.nativeElement;
+    const offsetDirection = this.orientation === 'horizontal' ? 'left' : 'top';
+
+    if (supportsScrollBehavior()) {
```

Comment:
Why do this differently based on `scroll-behavior`? Wouldn't that option just be ignored if it's not supported?

Reply:
The method signatures aren't compatible, old browsers only support:
`scrollTo(top: number, left: number)`

The new ones also support:
`scrollTo({top?: number, left?: number, behavior?: ScrollBehavior})`